include/zephyr/drivers/sensor/wsen_hids_2525020210002.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023 Würth Elektronik eiSos GmbH & Co. KG
```

Comment:
as a new file, shouldn't this be 2024?

Reply:
Yes guess i missed this one will fix now and tag you both again once its done